### DISCUSSION TITLE
Unable to compile with display

### DISCUSSION THREAD

**Discussion by FrankHovis** [luc-github/ESP3D#985] [F.A.Q]
> esp3d/src/core/esp3doutput.cpp:64:43: error: 'DEFAULT_SCREEN_FLAG' was not declared in this scope
 uint8_t ESP3DOutput::_screenoutputflags = DEFAULT_SCREEN_FLAG;
                                           ^~~~~~~~~~~~~~~~~~~

- Any suggestions?

**Comment by luc-github** [comment]
not with such few informations of what you are doing
